deimos/openssl/evp.d

```diff
@@ -553,6 +553,10 @@ void	EVP_MD_CTX_init(EVP_MD_CTX* ctx);
 int	EVP_MD_CTX_cleanup(EVP_MD_CTX* ctx);
 EVP_MD_CTX* EVP_MD_CTX_create();
 void	EVP_MD_CTX_destroy(EVP_MD_CTX* ctx);
+EVP_MD_CTX* EVP_MD_CTX_new();
```

Comment:
`EVP_MD_CTX_new` was added (renamed from `EVP_MD_CTX_create`) in OpenSSL 1.1.0.  https://www.openssl.org/docs/man1.1.0/man3/EVP_DigestInit.html
Should we add `static if (OPENSSL_VERSION_AT_LEAST(1, 1)` here?

Reply:
Ah, I got that `OPENSSL_VERSION_AT_LEAST(1, 1)` returns always true. Why this function exists?